vllm/envs.py

```diff
@@ -157,7 +157,7 @@
     VLLM_USE_FLASHINFER_MOE_FP16: bool = False
     VLLM_USE_FLASHINFER_MOE_FP8: bool = False
     VLLM_USE_FLASHINFER_MOE_FP4: bool = False
-    VLLM_FLASHINFER_MOE_BACKEND: Literal["throughput", "latency"] = "latency"
+    VLLM_FLASHINFER_MOE_BACKEND: Literal["throughput", "latency", "cutedsl"] = "latency"
```

Comment:
Is cutedsl supposed to be used in a high throughput or low latency situation?  I feel like the name could be more descriptive.

Reply:
cutedsl is a backend name and throughput implies flashinfer_cutlass_moe and latency for flashinfer_trtllm_gen_moe. The name itself doesn't indicated hl or ll. cutedsl is mostly suitable for PD disagg case.